doc/tags/for.rst

```diff
@@ -66,14 +66,52 @@ Variable              Description
 ``loop.last``         True if last iteration
 ``loop.length``       The number of items in the sequence
 ``loop.parent``       The parent context
-===================== =============================================================
+``loop.cycle``        Cycle over a sequence of values
+``loop.changed``      True if previously called with a different value or if not called yet
+``loop.previous``     The value from the previous iteration (``null`` for the first iteration)
+``loop.next``         The value from the next iteration (``null`` for the first iteration)
+===================== ========================================================================
 
 .. code-block:: twig
 
     {% for user in users %}
         {{ loop.index }} - {{ user.username }}
     {% endfor %}
 
+Use ``loop.cycle`` to cycle among a list of values:
+
+.. code-block:: html+twig
+
+    {% for row in rows %}
+        <li class="{{ loop.cycle('odd', 'even') }}">{{ row }}</li>
+    {% endfor %}
+
+Use ``loop.previous`` and ``loop.next`` to compare the current value with the
+previous or next values:
+
+.. code-block:: twig
+
+    {% for value in values %}
+        {% if not loop.first and value > loop.previous %}
+            The value just increased!
+        {% endif %}
+        {{ value }}
+        {% if not loop.last and loop.next > value %}
+            The value will increase even more!
+        {% endif %}
+    {% endfor %}
+
+Use ``loop.changed`` to check if the value has changed since the last iteration:
```

Comment:
`since the last call` technically (it is totally possible not to call that for each iteration, or to call it multiple times in the same iteration, even though this is not the expected usage)

Reply:
True